README.md

```diff
@@ -116,6 +117,7 @@ In summary, the supported configurations for transports are:
 | Crazyflie 2.1      |         -          | Custom Radio Link |         -          |
 | Espressif ESP32    |         -          |  UART, WiFI UDP   |         -          |
 | ST Nucleo F446ZE <sup>1</sup> |         -          |       UART        |         -          |
+| ST Nucleo H743ZI <sup>1</sup> |         -          |       -        |         UART          |
```

Comment:
Is it possible for you to test the Ethernet compatibility? Is this board's networking interfaces supported by Zephyr? If so, it should no be so difficult to add networking transports.

Reply:
Working on this now, I'll put in a new PR when I get it working

Reply:
Awesome @CLansdaleBEC, thanks!